docs/articles/documentation/guides/concepts/typescript-and-coffeescript.md

```diff
@@ -47,47 +46,46 @@ When you run a test, Testcafe will output if there are any compilation errors.
 
 ### Customize Compiler Options
 
-TestCafe allows you to specify [TypeScript compiler options](https://www.typescriptlang.org/docs/handbook/compiler-options.html) in the `tsconfig.json` file. You can use these options to enable the JSX compilation, set aliases to React typings, or customize other compiler settings.
+TestCafe users can modify the settings of the [TypeScript compiler](https://www.typescriptlang.org/docs/handbook/compiler-options.html) to enable features such as JSX compilation, and support for React type aliases.
```

Comment:
When you say the `modify`, you need to show here the option that can be modified.
So, add link to the modified settings:

```js
TestCafe users can modify the [settings](link-to-the-default-typescript-compilation-settings)
```

Reply:
Add a note

```js
At present, you can customize options only for TypeScript compiler.
```

Reply:
AFAIK we do not 'advertise' future features in documentation, so there's no need to phrase it like that. Right now this compiler setting feature does not come across as a generic because the article only ever mentions TypeScript. I think that users are only going to be more confused if they see a statement such as the one you propose.

Reply:
The 'compiler options' feature is a generic interface to specify compiler options for all supported compilers: `es-next`, `typescript` and `flow`. At present, the customer can specify options only for TypeScript compiler. In the future, this method will be used to specify options for other compilers.

@VasilyStrelyaev

Reply:
True, but this is the `typescript-and-coffeescript.md` topic, so I don't think that we ought to say about other compilers.
When we support more, we'll have a different guide about compilers in general.

Although, what Mikhail says applies to the reference. It's good to introduce the `compilerOptions` flag/property as the place where we can set the compiler options in general, and then note that it currently supports TypeScript.